src/utils/cutPlane.js

```diff
@@ -89,3 +90,32 @@ export function removePlanes(viewer) {
   }
 }
 
+
+/**
+ * helper method to get the location of all of the planes currently in the scene
+ *
+ * @param {object} viewer
+ * @param {object} model
+ */
+export function addPlaneLocationToUrl(viewer, ifcModel) {
+  const PLANE_PREFIX = 'p'
+  if (viewer.clipper.planes.length > 0) {
+    let planeNormal
+    let planeAxisCenter
+    let planeOffsetFromCenter
+    let planeHash
+    const planeOffsetFromModelBoundary = viewer.clipper.planes[0].plane.constant
+    const modelCenter = getModelCenter(ifcModel)
+    for (const [key, value] of Object.entries(viewer.clipper.planes[0].plane.normal)) {
+      if (value !== 0 ) {
+        planeNormal = key
+        planeAxisCenter = modelCenter[planeNormal]
+        planeOffsetFromCenter = planeOffsetFromModelBoundary - planeAxisCenter
+        const planeOffsetFromCenterTrim = planeOffsetFromCenter
+        planeHash = `${planeNormal},${planeOffsetFromCenterTrim}`
```

Comment:
So I think we want to reserve comma for lists of params in general (like we have in camera), and here when we have multiple planes.

I put up a syntax spec on the wiki:

https://github.com/bldrs-ai/Share/wiki/Design:-URL-Structure#param-syntax

That work?  So change the "," here to "=".

Reply:
yep

Reply:
Looks like this still needs to be resolved.